src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((block) => block.trim())
+    .filter((block) => block.includes(':'))
+    .reduce((cssProperties, block) => {
+      const [key, value] = block
+        .split(':')
+        .map((splitBlock) => splitBlock.trim());
+
+      cssProperties[key] = value;
```

Comment:
Directly assigning properties to the `cssProperties` object within the `reduce` method is acceptable, but ensure that the key names are sanitized to handle potential edge cases, such as keys with extra whitespace or special characters that may not be valid object property names.